lib/shopify-cli/app_types/node.rb

```diff
@@ -95,7 +95,7 @@ def check_npm_node
           dep_name = dep.split.first
           dep_link = dep_name == 'node' ? 'https://nodejs.org/en/download.' : 'https://www.npmjs.com/get-npm'
           version, stat = ctx.capture2e(dep)
-          ctx.puts("{{green:✔︎}} #{dep_name} #{version}")
+          ctx.puts("{{v}} #{dep_name} #{version}")
```

Comment:
is that a `v` or a checkmark?

Reply:
that is a v! 
that's what the ui kit uses for checkmarks
`CHECK    = new('v', 0x2713,  Color::GREEN)`

Reply:
oh, sweeeet